src/PowerShellEditorServices/Session/RemoteFileManager.cs

```diff
@@ -74,16 +74,22 @@ public class RemoteFileManager
 
             Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile {0}
 
-            if ((Test-Path -Path 'function:\global:PSEdit') -eq $false)
+            if ((Test-Path -Path 'function:\global:Open-EditorFile') -eq $false)
             {{
-                Set-Item -Path 'function:\global:PSEdit' -Value $PSEditFunction
+                Set-Item -Path 'function:\global:Open-EditorFile' -Value $PSEditFunction
+                Set-Alias psedit Open-EditorFile -Scope Global
             }}
         ";
 
         private const string RemovePSEditFunctionScript = @"
-            if ((Test-Path -Path 'function:\global:PSEdit') -eq $true)
+            if (Test-Path -Path 'function:\global:Open-EditorFile')
             {
-                Remove-Item -Path 'function:\global:PSEdit' -Force
+                Remove-Item -Path 'function:\global:Open-EditorFile' -Force
+            }
+
+            if (Test-Path -Path 'alias:\psedit')
+            {
+                Remove-Item -Path 'alias:\psedit' -Force
             }
 
             Get-EventSubscriber -SourceIdentifier PSESRemoteSessionOpenFile -EA Ignore | Remove-Event
```

Comment:
This PR may not be the right place to change this, but are we sure `Remove-Event` actually works here?  I feel like either that's supposed to be `Unregister-Event` or `Get-EventSubscriber` is supposed to be `Get-Event`

Reply:
Honestly I'm not sure, I stole this code from the ISE codebase :)  If you think there's an actual problem here we should definitely fix it

Reply:
Yep

```powershell
Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile -Forward
Get-EventSubscriber -SourceIdentifier PSESRemoteSessionOpenFile | Remove-Event

# Remove-Event : The input object cannot be bound to any parameters for the command either because the command does not
# take pipeline input or the input and its properties do not match any of the parameters that take pipeline input.
# At line:1 char:65
# + ... ntSubscriber -SourceIdentifier PSESRemoteSessionOpenFile | Remove-Event
# +                                                              ~~~~~~~~~~~~
#     + CategoryInfo          : InvalidArgument: (System.Manageme...EventSubscriber:PSObject) [Remove-Event], ParameterB
#    indingException
#     + FullyQualifiedErrorId : InputObjectNotBound,Microsoft.PowerShell.Commands.RemoveEventCommand
```

@tylerl0706  `Remove-Event` should be `Unregister-Event`

Reply:
You are right :) 

```
PS > Get-EventSubscriber | Remove-Event
Remove-Event : The input object cannot be bound to any parameters for the command either because the command does not take pipeline input or the input and its properties do not match any of the parameters that take pipeline input.
At line:1 char:23
+ Get-EventSubscriber | Remove-Event
+                       ~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (System.Manageme...EventSubscriber:PSObject) [Remove-Event], ParameterBindingException
+ FullyQualifiedErrorId : InputObjectNotBound,Microsoft.PowerShell.Commands.RemoveEventCommand

PS > Get-EventSubscriber | Unregister-Event
PS >
```

I'll change it to `Unregister-Event`.

Reply:
Done :)